frontend/public/locales/en/translation.json

```diff
@@ -562,10 +562,15 @@
     "creation.app.section.prePost": "Configure automation for prehook and posthook",
     "creation.app.section.settings": "Settings: Specify application behavior",
     "creation.app.section.techPreview": "Technology Preview",
-    "creation.app.select.existing.placement.rule": "Select existing placement rule",
+    "creation.app.select.existing.placement.rule": "Select an existing placement configuration",
+    "creation.app.settings.clusterSelector": "Create a placement configuration to deploy on clusters with specified labels",
+    "creation.app.local.clusters.only": "{{0}} deploys only to local cluster",
+    "creation.app.clusters.matching": "{{0}} deploys to clusters clusters with these labels: {{1}}",
+    "creation.app.clusters.all.online": "{{0}} deploys to any online clusters and local cluster",
```

Comment:
There should be 2 versions of this similar to `creation.app.settings.onlineClustersOnly` and `creation.app.settings.onlineClusters` that switch based on whether hub self management is enabled.

Reply:
And "any" should be "all"

Reply:
First comment not addressed.

Reply:
Fixed now.